.tool-versions

```diff
@@ -0,0 +1,2 @@
+pdm 2.10.4
+python 3.11.6
```

Comment:
What does the `.tool-versions` file do?

Reply:
it's for https://asdf-vm.com/

Reply:
I'm going to go ahead and leave this out since I don't use it and I probably won't maintain it!